src/axislike.rs

```diff
@@ -688,3 +703,22 @@ impl From<DualAxisData> for Vec2 {
         data.xy
     }
 }
+
+/// The shape of the deadzone for a [`DualAxis`] input.
+/// 
+/// Uses the x and y thresholds on [`DualAxis`] to set the shape diamensions
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, Hash)]
+pub enum DeadzoneShape{
+    /// Deadzone with a cross
```

Comment:
These docs need some improvement: what does the deadzone actually *look* like?

Reply:
This hard to put into words as it easier to see with an image but, `Cross` is two intersecting rectangles that have can have their widths changed independently. Is that clear, or does it still need some work shopping?

Reply:
Hmm, still not clear to me. Eventually I'd like to have an interactive example that draws them, but that can wait for a new PR.